src/utils/Regex.ts

```
@@ -1,29 +1,37 @@
   rewriteContext?: RewriteKeyContext,
-  scopes: ScopeRange[] = [],
+  scopes: ScopeRange[] | NextIntlScopeRange[] = [],
   namespaceDelimiters = [':', '/'],
   defaultNamespace?: string,
   starts: number[] = [],
 ): KeyInDocument | undefined {
   const matchString = match[0]
-  let key = match[1]
+
+  let keyIndex = 1
+  let nextIntlFunctionName = undefined
```

ESLint

[error] It's not necessary to initialize 'nextIntlFunctionName' to undefined.

(no-undef-init)